_events/2015-09-25.md

```diff
@@ -0,0 +1,22 @@
+---
+title: "Music Education Hackathon"
+description: "Music Education Hackathon"
```

Comment:
I know these are old events so it doesn't matter much but the description doesn't add much. On the Past Events page, it will say

**Music Education Hackathon**
Music Education Hackathon

which isn't great...